invoice/src/main/java/org/killbill/billing/invoice/tree/NodeInterval.java

```diff
@@ -179,7 +180,8 @@ public boolean addNode(final NodeInterval newNode, final AddNodeCallback callbac
         return insertNode(this, prevChild, null, newNode, callback);
     }
 
-    public void removeChild(final NodeInterval toBeRemoved) {
+    @VisibleForTesting
+    void removeChild(final NodeInterval toBeRemoved) {
```

Comment:
Same question: it looks like it's only used in `testRemove*` which test it.

Reply:
See https://github.com/killbill/killbill/pull/1493/commits/0509ee20a4f8cd2ea83910d6ea4f3a566f970548